tests/flexalloc_rt_strp_object_read_write.c

```diff
@@ -62,7 +62,8 @@ test_strp(struct test_vals test_vals)
 
   if (dev._is_zns)
   {
-    test_vals.slab_nlb = dev.nsect_zn;
+    // *2 to give it extra room to execute
```

Comment:
This doesn't make sense to me, and sounds like a hack (?).
If not, can you write a longer comment explaining the rationale here. E.g. what needs "room to execute"? Why a factor of 2?

Reply:
For the tests I need one striped object. But I just cant leave it exactly as one striped object because the slab has metadata that it has to fit in that space. So I have one of two choices: 1. Calculate metadata size and add that (hard), 2. Take out the big hammer and double the size.

I'll add a comment